lib/msal-common/src/client/AuthorizationCodeClient.ts

```diff
@@ -266,6 +279,17 @@ export class AuthorizationCodeClient extends BaseClient {
             }
         }
 
+        if (request.tokenBodyParameters) {
+            parameterBuilder.addExtraQueryParameters(request.tokenBodyParameters);
+        }
+
+        // Add hybrid spa parameters if not already provided
+        if (request.enableSpaAuthCode && (!request.tokenBodyParameters || request.tokenBodyParameters && !request.tokenBodyParameters[AADServerParamKeys.RETURN_SPA_CODE])) {
```

Comment:
Shouldn't need to assert truthiness again
```suggestion
        if (request.enableSpaAuthCode && (!request.tokenBodyParameters || !request.tokenBodyParameters[AADServerParamKeys.RETURN_SPA_CODE])) {
```

Reply:
Indeed, thanks